yt_dlp/networking/_requests.py

```diff
@@ -255,7 +255,8 @@ def __init__(self, *args, **kwargs):
         handler.setFormatter(logging.Formatter('requests: %(message)s'))
         handler.addFilter(Urllib3LoggingFilter())
         logger.addHandler(handler)
-        logger.setLevel(logging.WARNING)
+        # Use ERROR to suppress pool reuse warning (could filter instead?)
+        logger.setLevel(logging.ERROR)
```

Comment:
```suggestion
        logger.setLevel(logging.ERROR)
```

don't think this needs a comment imo. was supposed to be like this from the start

Reply:
I added a comment since we might want to explore the filter option instead of setting to `ERROR`

Reply:
I'll make it be a TODO to use a filter instead then